js/resize.js

```diff
@@ -8,13 +8,34 @@ function resize() {
     waterUseViz.viewport = 'narrow';
   }
   
-  try {
-   document.createEvent("TouchEvent");
-   waterUseViz.interactionMode = 'tap';
-  }
-  catch (e) {
+ var result = false;
+ 
+ if (window.PointerEvent && ('maxTouchPoints' in navigator)) {
+   // if Pointer Events are supported, just check maxTouchPoints
+   if (navigator.maxTouchPoints > 0) {
+     result = true;
+   }
+ } else {
+   // no Pointer Events...
+   if (window.matchMedia && window.matchMedia("(any-pointer:coarse)").matches) {
+     // check for any-pointer:coarse which mostly means touchscreen
+     result = true;
+   } else if (window.TouchEvent || ('ontouchstart' in window)) {
+     // last resort - check for exposed touch events API / event handler
+     result = true;
+   }
+ }
+ 
+ if(result === true){
+   if(screen.width >= 768 && screen.height >= 1024){
```

Comment:
I had worked myself around to 
```
if(Math.max(screen.width, screen.height) > 1024)
```

We need > rather than >= so we don't include assume hovering on ipads, and i think doing it as a max of height & width means we don't need to worry about which orientation it's in.

Are you persuaded by that? happy to hear an argument for keeping what you have if you're confident in it.